text/0000-cbool.md

```diff
@@ -0,0 +1,28 @@
+- Feature Name: cbool
+- Start Date: Mon Mar  9 00:16:53 CET 2015
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Specify that `bool` is compatible with the `_Bool` C type.
+
+# Motivation
+
+You cannot safely call ffi functions with boolean arguments without a compatible type.
+
+# Detailed design
+
+Specify that `bool` is compatible with the `_Bool` C type and inform the compiler that `bool` is ffi safe.
+
+# Drawbacks
+
+None right now. This should be a no-op on all supported platforms.
```

Comment:
>as true as possible

Even though it cannot be expressed in current rust ffi code, "true" implies a binary relation, so "as true as possible" does not make much sense.

It is true that `i8` is one byte on all supported platforms.